noise/src/state.rs

```diff
@@ -271,6 +271,7 @@ impl<E: Ecdh, D: Digest> HandshakeState<E, D> {
         let curve_name = match E::Pk::CURVE_NAME {
             "Curve25519" => "25519",
             "Secp356k1" => "secp256k1",
```

Comment:
Typo here FYI

Reply:
Sorry, where? `secp256k1` with lowercase letter is a typo by Lightning Labs, not be: that's the name they are using for the Noise handshake in Lightning (the only implementation using Secp256k1 for Noise I am aware of).

Reply:
Can you pls clarify this point so I can merge or fix this

Reply:
It says "Secp_3_56k1"

Reply:
shit, thank you for spotting that out!!!